packages/iframe-coordinator/src/elements/frame-router.ts

```diff
@@ -152,6 +162,24 @@ export default class FrameRouterElement extends HTMLElement {
     }
   }
 
+  private _frameSetup: FrameSetup;
+
+  /**
+   * A property that can be set to initialize the host frame
+   */
+  get frameSetup(): any {
```

Comment:
`frameSetup` feels like a bit of an awkward name for a property, although I can see the motivation in mirroring the method name. In hindsight, the method name isn't very good, as we're not really setting up the "frames", we're configuring the client routing and init. Also there's only actually one frame.  I would call this option something like `clientConfiguration` or `clientConfig`.  Maybe we even mark `setupFrames()` as deprecated when the 5.0 release goes out of beta.